### ISSUE TITLE
Entry Point Not Found

### ISSUE THREAD

**Issue by jbroski** [BroVic/RQDAassist#15]
I'm trying to install RQDAassist. 
I'm running R.4.1.2.
I've installed RTools and received verification that  "make" could be found (something to do with the .Renviron file. I don't know how it works, just know that these are the instructions to install RTools 4.0.

When I run remotes::install_github("BroVic/RQDAassist")
I get just past the step where stringi is successfully unpacked but then I get a pop-up box message:

     The procedure entry point EXTPTR_PROT could not be located in the dynamic link library.

Do you have any recommendations about how to address this?

Julie

P.S. Thank you for posting this package and for your help assisting users get up to speed. I've been a long time user of RQDA and I truly appreciate this effort!

**Comment by BroVic** [comment]
Hello @jbroski 

The error you're encountering has something to do with the tool chain, so it's between R and rtools40. My suggestion is you try reinstalling them and while you're at it, upgrading to R 4.1 3. If it still fails, let me know so we can run a more detailed search for a solution. When installing rtools40 avoid messing with any configurations but check that both it and the version of R you're using are on `PATH`.

P.S.: 
Thanks for the encouraging words.

**Comment by jbroski** [comment]
I get the verification that rtools is on PATH. (I don't know how to check or ensure that the version of R I'm using is on PATH.)

However, I'm now getting a new/different error when I try to install RQDAassist:

Error in library.dynam(lib, package, package.lib) : 
  DLL ‘fs’ not found: maybe not installed for this architecture?

Any recommendations are welcome!

**Comment by BroVic** [comment]
Could you please share the output from `sessionInfo()`?

**Comment by jbroski** [comment]
Sure.

> sessionInfo()
R version 4.1.3 (2022-03-10)
Platform: i386-w64-mingw32/i386 (32-bit)
Running under: Windows 10 x64 (build 22000)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252 LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_4.1.3 tools_4.1.3

**Comment by BroVic** [comment]
I recommend you force the installation of binaries instead of the source version of the package(s) that are problematic. For example for **fs** (the package raising the last error, do

```
install.packages("fs", type = "binary")
```

It's a bit hacky, but let's if it works for you. There's a problem with the tool chain still and that's probably why you can't compile certain packages.

**Comment by BroVic** [comment]
@jbroski I hope you've been able to make progress...

**Comment by jbroski** [comment]
@BroVic I was able to install the fs package. Yesterday I received a message that there was a problem with memory,. I dug around on the internet to find a solution, to increase memory. For some reason it didn't work yesterday, but today it worked. 

Today after installing fs as a binary package I was able to move forward from the previous error, but then received this error:

Error in inDL(x, as.logical(local), as.logical(now), ...) : 
  unable to load shared object 'C:/Users/julie/OneDrive/Documents/R/win-library/4.1/rlang/libs/x64/rlang.dll':
  LoadLibrary failure:  The specified procedure could not be found.

I tried adding the rlang package manually and I received the message that it was successfully unpacked
package ‘rlang’ successfully unpacked and MD5 sums checked.

I tried installing RQDAassist again and received this message that rdb' is corrupt
> devtools::install_github("BroVic/RQDAassist")
Error in get(Info[i, 1], envir = env) : 
  lazy-load database 'C:/Users/julie/OneDrive/Documents/R/win-library/4.1/rlang/R/rlang.rdb' is corrupt
In addition: Warning message:
In get(Info[i, 1], envir = env) : internal error -3 in R_decompress1
